docs/rules/jsx-first-prop-new-line.md

```diff
@@ -69,6 +75,27 @@ The following patterns are not considered warnings when configured `"multiline"`
 />
 ```
 
+The following patterns are considered warnings when configured `"multiline-multiprop"`:
+
+```jsx
+<Hello foo={{
+    }}
+    bar />
+```
+
+The following patterns are not considered warnings when configured `"multiline-multiprop"`:
+
+```jsx
+<Hello foo={{
```

Comment:
hmm, i misunderstood the purpose of this option - it seems like Airbnb's config already matches `multiline`, and we wouldn't use this particular option.

This thus seems like it does match what you're looking for.